stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/response/AppSettingsAPIResponse.kt

```diff
@@ -0,0 +1,32 @@
+package io.getstream.chat.android.client.api2.model.response
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+public data class AppSettingsAPIResponse(
```

Comment:
All of those classes can be internal, right?
Also, instead of using `@Json(name = "")` let's match parameters names with those from json as we do in other cases. In API response-related models we are using snake_case. Check `DownstreamChannelDto` for reference